python/labbox_ephys/workspace/workspace.py

```diff
@@ -6,9 +6,9 @@ class Workspace:
     def __init__(self, *, feed: kp.Feed, workspace_name: str) -> None:
         self._feed = feed
         self._workspace_name = workspace_name
-        recordings_subfeed = self._feed.get_subfeed(dict(workspaceName=self._workspace_name, key='recordings'))
+        recordings_subfeed = self._feed.get_subfeed(dict(workspaceName=self._workspace_name))
```

Comment:
For simplicity, we now have a single subfeed for the workspace, rather than separate subfeeds for recordings/sortings. It simplifies the interaction between client/server quite a bit.